docs/data/projects.yml

```diff
@@ -30,8 +36,8 @@
 
 - name: Parselmouth
   gh: YannickJadoul/Parselmouth
-  pypi: praat-parselmouth
-  notes: A Python interface to the Praat software package, using pybind11, C++17 and CMake, with the core Praat static library built only once and shared between wheels.
+  pypi: praat-parselmouthA Python interface to the Praat software package, using pybind
+  notes: 11, C++17 and CMake, with the core Praat static library built only once and shared between wheels.
```

Comment:
Could these changes be reverted please?

Reply:
I have not made these changes, and I believe they can be reverted.

Reply:
Would you mind reverting them? They're in your PR, so I suppose that you made them by accident.

Reply:
No problem. I going to revert them.

Reply:
I believe they have been reverted. Can you check it?

Reply:
LGTM